ticket_facts.view.lkml

```diff
@@ -0,0 +1,277 @@
+      description: "Yes, if inbound email was solved and has outbound activity"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} LIKE 'Inbound Email%'
+          AND ${status} IN ('solved', 'closed')
+          AND ${is_ticket_merged} = false
+          AND (${number_outbound_emails} + ${number_outbound_calls}) >= 1 ;;
+    }
+
+    dimension: is_first_touch_resolution_phone_call {
+      description: "Yes, if inbound phone call was solved at first agent touch"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} IN ('Inbound Call','Inbound Voicemail')
+           AND ${status} IN ('solved', 'closed')
+           AND ${is_ticket_merged} = false
+           AND ${number_inbound_calls} = 1
+           AND (${number_outbound_emails} + ${number_outbound_calls}) = 0
+           AND ${is_parent_to_merged_tickets} = false ;;
+    }
+
+    dimension: is_eligible_for_first_touch_resolution_phone_call {
+      description: "Yes, if inbound phone call was solved"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${ticket_source} IN ('Inbound Call','Inbound Voicemail')
+           AND ${status} IN ('solved', 'closed')
+           AND ${is_ticket_merged} = false
+           AND ${number_inbound_calls} >= 1 ;;
+    }
+
+    dimension: number_outbound_emails {
+      description: "Number of outbound emails associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_outbound_emails ;;
+    }
+
+    dimension: has_outbound_emails {
+      description: "Yes, if ticket has associated outbound emails"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_outbound_emails} > 0 ;;
+    }
+
+    dimension: number_internal_comments {
+      description: "Number of internal comments associated with ticket"
+      group_label: "Activity Counts"
+      type: number
+      sql: ${TABLE}.number_internal_comments ;;
+    }
+
+    dimension: has_internal_comments {
+      description: "Yes, if ticket has internal comments created"
+      group_label: "Activity Details"
+      type: yesno
+      sql: ${number_internal_comments} > 0 ;;
+    }
+
+    dimension: is_parent_to_merged_tickets {
+      description: "Yes, if the ticket is the parent to other merged tickets"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${TABLE}.number_merged_tickets > 0 ;;
+    }
+
+    dimension: is_ticket_merged {
+      description: "Yes, if the ticket has been merged into another ticket"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${TABLE}.is_ticket_merged > 0 ;;
+    }
+
+    dimension: is_automatically_generated {
+      description: "Yes, if ticket has been created automatically.  Generally via Sentinel"
+      group_label: "Ticket Details"
+      type: yesno
+      sql: ${TABLE}.is_auto_created > 0 ;;
+    }
+
+    measure: count {
```

Comment:
Should we add sum fields for all your `number_*` dimensions?

Reply:
I haven't found a need for it yet, so going to hold off for now.  It might be helpful for looking at a true total of calls or tickets made by a team, but let's save that for when we have to set up our new Stitch/ZD tables.